yaml/src/main/yaml/KafkaToBigQuery.yaml

```diff
@@ -38,46 +45,52 @@ template:
       type: integer
 
     - name: "storageWriteApiTriggeringFrequencySec"
-      help: "Triggering frequency in seconds for BigQuery Storage Write API"
+      help: "Triggering frequency will determine how soon the data will be visible for querying in BigQuery."
       required: false
-      default: 3
+      default: 1
```

Comment:
Where did this default come from? I don't necessarily have a better one offhand, though my default would be to copy our docs and do 5 - https://cloud.google.com/dataflow/docs/guides/write-to-bigquery#stream-data

If there is a good reason, feel free to ignore.

Reply:
Good catch.  Can't remember why I set to 1.  Will change to 5.  Thanks.